### PR TITLE
device: debian: retrieves a file form the tftp to the localhost

### PR DESCRIPTION
**Pull request by mgualco-contractor**
retrieves a file form the tftp server to the localhost.
Signed-off-by: Michele Gualco <mgualco.contractor@libertyglobal.com>

### CODE REVIEW

File: devices/debian.py [hunk 1]
```diff
@@ -402,6 +402,41 @@ def start_sshd_server(self):
         self.sendline('/etc/init.d/ssh reload')
         self.expect(self.prompt)
 
+    def get_tftp_cfg_dest_path(self, board=None):
+        return '/tftpboot' # can this change?
```
Comment (mattsm) [inline devices/debian.py]: Not so far, but we can just make this an arg in the function below with the default being /tftpboot for now?

File: devices/debian.py [hunk 2]
```diff
@@ -402,6 +402,41 @@ def start_sshd_server(self):
...
+
+    def get_conf_file_from_tftp(self, board, cfg):
+        """Retrieve a file from the tftp sever"""
+        if cfg is None:
+            return None
```
Comment (mattsm) [inline devices/debian.py]: Can we make the default action to download the CM and MTA cfg files? I assume this runs on the provisioner? We should add a function there to return the current configs (as read from DHCP config), so we can just use those filenames/paths

File: devices/debian.py (hunk 1 above)
Comment (mgualco-contractor) [reply devices/debian.py]: ACK

File: devices/debian.py (hunk 2 above)
Comment (mgualco-contractor) [reply devices/debian.py]: yes, we can, but wouldn't it be easier to get them directly from the device /var/tmp?

**Comment by mattsm** [comment]
We talked more about this- any more questions?

**Comment by mgualco-contractor** [comment]
they will come....

**Pull request merged**